disruptor/src/main/java/org/axonframework/disruptor/commandhandling/DisruptorCommandBus.java

```diff
@@ -240,10 +243,9 @@ protected DisruptorCommandBus(Builder builder) {
         publisherCount = publishers.length;
         messageMonitor = builder.messageMonitor;
         duplicateCommandHandlerResolver = builder.duplicateCommandHandlerResolver;
-
         disruptor = new Disruptor<>(CommandHandlingEntry::new,
```

Comment:
The `Disruptor` constructor taking in an `Executor` is deprecated in favor of providing a `ThreadFactory`.
Decided to make the `ThreadFactory` part of this command bus' builder, so that users can provide their own for the `Disruptor` if desired.
Defaulted to the `AxonThreadFactory`, as we defaulted to this when no `Executor` was given too.